test/callback.cc

```diff
@@ -3,8 +3,8 @@
 #include "gtest/gtest.h"
 #include "stout/borrowed_ptr.h"
 
-using eventuals::Callback;
-
+namespace eventuals {
+namespace {
 TEST(Callback, Destructor) {
```

Comment:
To stay consistent with the style in the code base please keep a newline between `namespace` and anything else!